docs/csharp/language-reference/builtin-types/reference-types.md

```diff
@@ -105,7 +159,7 @@ Console.WriteLine(a);
 @"good morning"  // a string literal
 ```
 
-The advantage of verbatim strings is that escape sequences are *not* processed, which makes it easy to write, for example, a fully qualified Windows file name:
+The advantage of verbatim strings is that escape sequences *aren't* processed, which makes it easy to write. For example, the following text matches a fully qualified Windows file name:
```

Comment:
personally i prefer teh non-abbreviated form here "are not".

Reply:
So do I, honestly. But the docs standard is to prefer the contractions.